plugin/build.gradle.kts

```diff
@@ -57,12 +58,14 @@ shadowJar.configure {
     }
 }
 
-tasks.getByName("jar").enabled = false
-tasks.getByName("jar").dependsOn(shadowJar)
+tasks.getByName("jar").apply {
+    enabled = false
+    dependsOn(shadowJar)
+}
 
 gradlePlugin {
     plugins {
-        create("testRetry") {
+        register("testRetry") {
```

Comment:
is it okay to make these lazy?

Reply:
I don't think it makes a difference here. IIRC they are eagerly created regardless.